platforms-maturity-model/v1/index.md

```diff
@@ -0,0 +1,515 @@
+|:---------------------------------------|:-------------------------------------------------------------------------------------------|:-----------------------|:----------------------|:-----------------------|:-----------------------------|
+| [Investment](#tab-text-Investment)     | _How are staff and funds allocated to platform capabilities?_                              | Voluntary or temporary | Dedicated team        | As product             | Enabled ecosystem            |
+| [Adoption](#tab-text-Adoption)         | _Why and how do users discover and use internal platforms and platform capabilities?_      | Erratic                | Extrinsic push        | Intrinsic pull         | Participatory                |
+| [Interfaces](#tab-text-Interfaces)     | _How do users interact with and consume platform capabilities?_                            | Custom processes       | Standard tooling      | Self-service solutions | Integrated services          |
+| [Operations](#tab-text-Operations)     | _How are platforms and their capabilities planned, prioritized, developed and maintained?_ | By request             | Centrally tracked     | Centrally enabled      | Managed services             |
+| [Measurement](#tab-text-Measurement)   | _What is the process for gathering and incorporating feedback and learning?_               | Ad hoc                 | Consistent collection | Insights               | Quantitative and qualitative |
+
+## Model Detail
+
+<div style="min-width:620px">
+{{< tabs tabTotal="5">}}
+{{< tab tabName="Investment" >}}
+
+</br>
+
+<h4 style="color:gray"><i>How are staff and funds allocated to platform capabilities?</i></h4>
+</br>
+
+Investment in platforms and platform engineering is the process of allocating budget and people to build and maintain common capabilities. It is common for initiatives to be described as organically built from the bottom up, or driven through top down initiatives. In either case, it is the ability to invest sustained effort that drives high-impact work. This aspect captures how the scale and breadth of investment can impact platform success.
+
+#### Level 1, Provisional - Voluntary or temporary
+
+Individual capabilities may exist to provide common foundations for common or critical functionality. These capabilities are built and maintained out of necessity rather than planned and intentionally funded. 
+
+These capabilities are built and maintained by people assigned temporarily or voluntarily; no central funding or staffing are intentionally allocated to them. They depend on the current tactical requirements of their users.
+
+##### Characteristics:
+
+* "Hit" or "tiger" teams are built to tackle urgent technical requirements. These teams are short lived and not assigned nor granted the time to provide long term planning and support.
```

Comment:
Do Tiger teams always tackle just technical requirements? Or Product requirements as well?

Reply:
I think removing `technical` would make more sense. I like that concept of a migration because a migration could be technical (migration from old software to new to improve technical) but you could also think of it as a tiger team migrating from one software to another to remove the need for a contract which would not be a technical requirement but maybe a business/cost requirement

Reply:
👍 That makes sense to me, incoming change to remove technical!